Gruntfile.coffee

```diff
@@ -29,15 +29,6 @@ module.exports = ->
             discover: true
             recursive: true
             silent: true
-          webpack:
```

Comment:
@bergie why does the existence of these keys break things? This is a breaking change in grunt-noflo-browser? Why was it not `2.0.0` to indicate?